lib/ansible/_internal/ansible_collections/ansible/_protomatter/README.md

```diff
@@ -0,0 +1,3 @@
+"Protomatter- an unstable substance which every ethical scientist in the galaxy has denounced as dangerously unpredictable."
+
+"But it was the only way to solve certain problems..."
```

Comment:
Good job on setting the mystery mood! It could be useful to have some explanatory prose in here too..